geometry_msgs/Point start
geometry_msgs/Point goal
---
geometry_msgs/Point[] path_points
std_msgs/Int32 explored_nodes
std_msgs/Int32 time_spent
std_msgs/Int32 line_of_sight_checks
std_msgs/Float32 path_length
